generated/thrift/rpc/rpc.go

```diff
@@ -28,7 +28,6 @@ import (
 	"database/sql/driver"
 	"errors"
 	"fmt"
-
```

Comment:
Any reason to remove this separation of go imports?  If not perhaps add it back for separation of stdlib imports and external imports.